sdk/metricsadvisor/Azure.AI.MetricsAdvisor/CHANGELOG.md

```diff
@@ -3,7 +3,12 @@
 ## 1.0.0-beta.5 (Unreleased)
 
 ### Breaking Changes
-- The whole `DatasourceCredential` API has been renamed to `DataSourceCredential`. This includes renames in types, methods, and properties.
+- The whole `DatasourceCredential` API has been renamed to `DataSourceCredential`. This includes renames in methods, method parameters, and properties.
+  - Renamed class `DatasourceCredential` to `DataSourceCredentialEntity`.
+  - Renamed class `DataLakeGen2SharedKeyDatasourceCredential` to `DataSourceDataLakeGen2SharedKey`.
```

Comment:
ohh cool. Just for my own learning, what was the reason for moving `DataSource` to the beginning of the types?

Reply:
We don't like ending in "Credential" that much because this can get mixed up with credential types in Identity. I'm okay with just appending "CredentialIdentity" to everything, but other languages are taking this approach, so I'm following along for consistency. The reasoning is something like "DataSourceSqlConnectionString" is a "data source's SQL connection string".

In terms of advantage, I can only think of:
- Types names get shorter (I've heard that `DataLakeGen2SharedKeyDataSourceCredential` is too long more than once).
- All credential types get organized in the same place because of alphabetical order.